File: CS50/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    float c;
    int i;
// Getting input
    do
    {
        c = get_float("Change owed: ");
    }
    while (c < 0);

    i = 0;
// 25 cents
    if (c >= 0.2495)
    {
        do
        {
            c = c - 0.25;
            i++;
        }
        while (c >= 0.2495);
    }
// 10 cents
    if (c >= 0.0995)
    {
        do
        {
            c = c - 0.10;
            i++;
        }
        while (c >= 0.0995);
    }
// 5 cents
    if (c >= 0.0495)
    {
        do
        {
            c = c - 0.05;
            i++;
        }
        while (c >= 0.0495);
    }
// 1 cent
    if (c >= 0.009)
    {
        do
        {
            c = c - 0.01;
            i++;
        }
        while (c >= 0.009);
    }
// Displaying result
    printf("%i\n", i);
}

File: CS50/pset6/cash/cash.py
from cs50 import get_float

# Getting input
while True:
    c = get_float("Change owed: ")
    if (c >= 0):
        break
c = c * 100
i = 0
# 25 cents
i = c // 25
c = c % 25
# 10 cents
i += c // 10
c = c % 10
# 5 cents
i = i + c // 5
c = c % 5
# 1 cent
i += c // 1
c = c % 1
# Displaying result
print(f"{i}")